src/main/java/Todo.java

```diff
@@ -0,0 +1,8 @@
+public class Todo extends Task {
+	public Todo (String info) {
+		super (info);
+	}
+	public String toString() {
```

Comment:
Perhaps adding override annotation to indicate that the method is overriding a method from the parent class. Be consistent in the annotations for all the classes.